File: src/main/java/domain/model/krkte/KrkteCalculator.java
package domain.model.krkte;

public class KrkteCalculator {
    private static final double MILLIS_PER_MINUTE = 1.6667e-5;

    public static double calculateKrkte(double airDensityGramsPerDecimetersCubed, double cylinderDisplacementDecimetersCubed, double fuelInjectorSizeCubicCentimeters, double gasolineGramsPerCubicCentimter, double stoichiometricAirFuelRatio) {
        return (airDensityGramsPerDecimetersCubed*cylinderDisplacementDecimetersCubed)/(100*MILLIS_PER_MINUTE*stoichiometricAirFuelRatio*fuelInjectorSizeCubicCentimeters*gasolineGramsPerCubicCentimter);
    }
}

File: src/main/kotlin/domain/model/krkte/KrkteCalculator.kt
package domain.model.krkte

object KrkteCalculator {
    private const val MILLIS_PER_MINUTE = 1.6667e-5

    fun calculateKrkte(
        airDensityGramsPerDecimetersCubed: Double,
        cylinderDisplacementDecimetersCubed: Double,
        fuelInjectorSizeCubicCentimeters: Double,
        gasolineGramsPerCubicCentimeter: Double,
        stoichiometricAirFuelRatio: Double
    ): Double {
        return (airDensityGramsPerDecimetersCubed * cylinderDisplacementDecimetersCubed) /
                (100 * MILLIS_PER_MINUTE * stoichiometricAirFuelRatio * fuelInjectorSizeCubicCentimeters * gasolineGramsPerCubicCentimeter)
    }
}
